.swiftlint.yml

```diff
@@ -0,0 +1,10 @@
+disabled_rules:
+  - trailing_whitespace
```

Comment:
Isn't this rule about a last empty line in the file? Or not having spaces at the end of a line?
I think it makes no harm to leave it and if it's in swiftlint it's because it's a good practice.